### PR TITLE
Removed deprecated method distributedobject getid

### PR DESCRIPTION
**Pull request by pveentjer**

### CODE REVIEW

**Comment by gurbuzali** [comment]
:+1:

File: hazelcast/src/main/java/com/hazelcast/core/DistributedObjectEvent.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public EventType getEventType() {
      * @return the identifier of DistributedObject
      */
     public Object getObjectId() {
```
Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/core/DistributedObjectEvent.java]: shouldn't this method be renamed/deprecated as well? 
as now there is no obvious connection between `eventObjectId` and `distributedObjectName`.

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/core/DistributedObjectEvent.java]: Let me see.

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/core/DistributedObjectEvent.java]: Good spotting. I have deprecated that method and added a getObjectName method.

**Comment by jerrinot** [comment]
:+1: (once it's green)

**Pull request merged**